docs/toolbar.md

```diff
@@ -0,0 +1,43 @@
+# Arduino Upload Package
+
+## Add a toolbar
+
+You can install `tool-bar` and `flex-tool-bar` Atom packages and use them to add a toolbar for Arduino commands, similar to the one in the official IDE.
+
+	apm install tool-bar
+	apm install flex-tool-bar
+	
+The following is an example for the Flex Tool Bar's configuration file, `toolbar.cson`. It adds the buttons for Arduino **Verify** and **Upload** commands (automatically saving the current file before), and another to open the **Serial Monitor**.
+
+	[
+	  {
+		type: "button"
+		icon: "check"
+		callback: ["core:save", "arduino-upload:verify"]
+		tooltip: "Arduino: Verify",
+		enable: { grammar: "arduino" }
+	  }
+	  {
+		type: "button"
+		icon: "arrow-right"
+		callback: ["core:save", "arduino-upload:upload"]
+		tooltip: "Arduino: Upload",
+		enable: { grammar: "arduino" }
+	  }
+	  {
+		type: "button"
+		icon: "terminal"
+		callback: "arduino-upload:serial-monitor"
+		tooltip: "Arduino: Serial monitor",
+		enable: { grammar: "arduino" }
+	  }
+	  {
+		type: "spacer"
+	  }
+	  {
+		type: "button"
+		icon: "gear"
+		callback: "flex-tool-bar:edit-config-file"
+		tooltip: "Edit Tool Bar"
+	  }
+	]
```

Comment:
This code would look nicer in a markdown code block with defined language 'coffee'

Reply:
Right, I added syntax highlighting for the CSON snippet. Nice idea about showing only on Arduino files, I just changed mine too, and I also added it to the last config button and to the spacer, so the toolbar goes completely away when I'm using Atom for other stuff (99% of the time). However I don't think it's good to put that in the example here, because that way the person who follows the instructions can't see the custom buttons appearing immediately, on the contrary, as soon as she saves the CSON she'll see the toolbar disappearing completely.